packages/salesforcedx-vscode-apex-replay-debugger/src/commands/quickLaunch.ts

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { LogService, TestService } from '@salesforce/apex-node';
+import {
+  ApexTestResultData,
+  SyncTestConfiguration,
+  TestItem,
+  TestResult
+} from '@salesforce/apex-node/lib/src/tests/types';
+import { Connection } from '@salesforce/core';
+import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { nls } from '../messages';
+import { getLogDirPath } from '../utils';
+import { launchFromLogFile } from './launchFromLogFile';
+import { TraceFlags } from './traceFlags';
+
+const sfdxCoreExports = vscode.extensions.getExtension(
+  'salesforce.salesforcedx-vscode-core'
+)!.exports;
+
+export const LibraryCommandletExecutor =
+  sfdxCoreExports.LibraryCommandletExecutor;
+export const notificationService = sfdxCoreExports.notificationService;
```

Comment:
Lets use the new notification service and workspace context on salesforcedx-utils-vscode module https://github.com/forcedotcom/salesforcedx-vscode/tree/develop/packages/salesforcedx-utils-vscode/src/context & https://github.com/forcedotcom/salesforcedx-vscode/tree/develop/packages/salesforcedx-utils-vscode/src/commands